test/selector_assertions_test.rb

```diff
@@ -214,46 +227,59 @@ def test_assert_select_text_match
   # Test assert_not_select.
   #
 
-  def test_not_select
+  def test_assert_not_select
     render_html '<div id="1"></div>'
     assert_not_select "p"
     assert_failure(/Expected exactly 0 elements matching "div", found 1/) { assert_not_select "div" }
     assert_failure(/Expected exactly 0 elements matching "div#1", found 1/) { assert_not_select "div#1" }
   end
 
-  def test_not_select_with_true
+  def test_assert_not_select_with_true
```

Comment:
These tests were added in #113 and I realised the naming is inconsistent with the other tests here, so I updated them.

Also added exception message assertions because there are now multiple `ArgumentError` branches and this ensures we're testing the correct scenario.

Reply:
Love it, thank you!